pkgs/development/libraries/gperftools/default.nix

```
@@ -28,8 +28,8 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ autoreconfHook ];
 
-  # tcmalloc uses libunwind in a way that works correctly only on non-ARM linux
-  buildInputs = lib.optional (stdenv.isLinux && !(stdenv.isAarch64 || stdenv.isAarch32)) libunwind;
+  # tcmalloc uses libunwind in a way that works correctly only on non-ARM dynamically linked linux
+  buildInputs = lib.optional (stdenv.isLinux && !(stdenv.isAarch64 || stdenv.isAarch32 || stdenv.hostPlatform.isStatic)) libunwind;
```

FYI

```suggestion
  buildInputs = lib.optional (stdenv.isLinux && !(stdenv.isAarch || stdenv.hostPlatform.isStatic)) libunwind;
```

after #183283 would be merged